src/routes/index.ts

```diff
@@ -3,11 +3,14 @@ export const router = express.Router()
 import registerController from "../controllers/registerController";
 import tokenMiddleware from "../middleware/tokenMiddleware";
 import loginController from "../controllers/loginController";
+import transactionController, {transactionHistory} from "../controllers/transactionController";
```

Comment:
If you make the change suggested above, you should be able to destructure both functions here